src/pages/Dashboard/widgets/BatchTransactions/hooks/useSendBatchTransactions.ts

```diff
@@ -0,0 +1,41 @@
+import { getBatchTransactions } from '../helpers';
+import { sendTransactions } from 'helpers/sdkDappHelpers';
+import { SessionEnum } from 'localConstants/session';
+import { SendTransactionProps } from '../types';
+
+export const useSendBatchTransactions = () => {
```

Comment:
why not function?

From docs: https://docs.multiversx.com/developers/guidelines/react-development#rules-for-hooks
If only 10% of the lines of code in a hook do specific React logic like state management, or calling a selector from Redux, consider splitting the hook into:
a smaller hook that does the React logic and returns the state, and
a statless function that will be called inside the hook and will do the rest of the logic. This stateless function should be tested separately.